README.md

```diff
@@ -1 +1,183 @@
-# Module_1_Hello_World
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="Kamran.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Hello World</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello World <a name="about-project"></a>
+
+> My first Hello World project for understanding of linters and github flow. I created it using HTML, CSS
+
+
+## 🛠 Built With <a name="built-with"></a>
+1- HTML.
+2- CSS.
+3- linters.
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/">HTML</a></li>
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Added index.html file
+- Added style.css file
+- Added text.md file
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Coming soon.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+1- Web browser.
+2- Code editor
+3- Git-smc
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Run this command:
+
+```sh
+  cd my-folder
+  git clone https://github.com/kamranasif506/Module_1_Hello_World.git
+```
+
+### Install
+
+Install this project with:
+
+Run this command:
+
+```sh
+  cd my-project
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+Open index.html file on live server extention.
+
+### Run tests
+
+> Coming soon.
+
+### Deployment
+
+> coming soon
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] Here as well, I would highly suggest you to please write appropriate content under each heading, so that your README file will look more professional and will give a better reading experience. For example, you consider adding the following information under the **Run tests** heading:

> You can run the tests using the following commands:
npx hint .
npx stylelint "**/*.{css,scss}"

And for **deployment**, you can mention how your project can be deployed, for example, using GitHub Pages, etc.